src/lib/cli.js

```diff
@@ -31,16 +31,34 @@ process.title = 'verdaccio';
 const logger = require('./logger');
 logger.setup(); // default setup
 
+const envinfo = require('envinfo');
 const commander = require('commander');
 const pkgVersion = module.exports.version;
 const pkgName = module.exports.name;
 
 commander
+  .option('--info', 'prints debugging information about the local environment')
   .option('-l, --listen <[host:]port>', 'host:port number to listen on (default: localhost:4873)')
   .option('-c, --config <config.yaml>', 'use this configuration file (default: ./config.yaml)')
   .version(pkgVersion)
   .parse(process.argv);
 
+if (commander.info) {
+	// eslint-disable-next-line no-console
+	console.log('\nEnvironment Info:');
+	envinfo
+	.run({
+	    System: ['OS', 'CPU'],
+	    Binaries: ['Node', 'Yarn', 'npm', 'docker'],
```

Comment:
Docker does not belong to `Binaries` according the docs. I think you rather need add `Virtualization` instead.
```
Binaries: ['Node', 'Yarn', 'npm'],
Virtualization: ['Docker'],
``

Reply:
Is it all lowercase or not relevant (case insensitive)?

Reply:
According the docs, seems to be case sensitive.

Reply:
Also the binary names and so on?
I see no consistency in this.